js/components/interface/menu/MenuSingleItem.js

```diff
@@ -95,8 +97,8 @@ class MenuSingleItem extends React.Component {
         style={appliedStyle}
         {...others}
       >
-        {item.icon 
-          ? <ListItemIcon>
+        {item.icon
+          ? <ListItemIcon style={ iconStyle }>
```

Comment:
@filippomc added this to make the component backward compatible with the previous style.

Reply:
Need to rethink in general the approach to the stlying to be compliant with material theming, but for now I think it's fine. Probably all the style directives in the configuration should be removed, as the menu should be consistent with the rest of the application rather than introducing fine grained styling on each element. For very custom stuff, we may leave the possibility to add a custom class on each item, so every application can do its stuff in a css file if needed.